scss/_custom-forms.scss

```diff
@@ -161,6 +161,48 @@
 }
 
 
+// switches
+//
+// Tweak a few things for switches
+
+.custom-switch {
+  padding-left: $custom-control-gutter * 1.5;
+
+  .custom-control-label {
+    &::before {
+      left: -$custom-control-gutter * 1.5;
+      width: $custom-switch-width;
+      pointer-events: all;
+      border-radius: $custom-switch-indicator-border-radius;
+    }
+
+    &::after {
+      top: calc(#{(($font-size-base * $line-height-base - $custom-control-indicator-size) / 2)} + #{$custom-control-indicator-border-width * 2});
+      left: calc(#{-$custom-control-gutter * 1.5} + #{$custom-control-indicator-border-width * 2});
+      width: $custom-switch-indicator-size;
+      height: $custom-switch-indicator-size;
+      background-color: $custom-control-indicator-border-color;
+      background-size: $custom-control-indicator-size;
+      border-radius: $custom-switch-indicator-border-radius;
```

Comment:
This is always `50%` I think?

Reply:
Forget this, you actually also took care of the ability to use switches that aren't rounded:
![image](https://user-images.githubusercontent.com/11559216/49306616-08efa500-f4d3-11e8-8a33-16aac5f3feee.png)

Sweet!